tests/main.test.ts

```diff
@@ -1384,13 +1384,14 @@ describe('zip-it-and-ship-it', () => {
   })
 
   testMany(
-    'Adds `type: "functionsBundling"` to user errors when parsing with esbuild',
-    ['bundler_esbuild'],
+    'Adds `type: "functionsBundling"` to user errors when parsing with esbuild or zisi',
+    ['bundler_esbuild', 'bundler_esbuild_zisi', 'bundler_default'],
```

Comment:
What happens with NFT?

Reply:
Not sure exactly, but I know it ignores missing files and probably also files that have a syntax error while finding imports. So it does not throw at buildtime.